### ISSUE TITLE
Incomplete Icon Set?

### ISSUE THREAD

**Issue by AidFTech** [B00merang-Artwork/Windows-7#9]
I'm looking to use this theme and icon set on my new Dell Latitude XT3 (which originally shipped with Windows 7 and now runs Linux Mint) and have noticed the same issue with the Mint icons on both the panel applets and the File Browser sidebar. My guess is that the mapping was different when this set was first drawn, and when Mint was updated, this set never was. Would it be possible for the set to be updated soon?

Most of the larger file and folder icons are correct. I think it's the "symbolic" ones (and some of the launcher icons) that are having issues.

Below: Top images are from a virtual machine running Windows 7, bottom image is from this machine with the Windows 7 theme and Mint icons.

Thank you!

![Correct Panel Icons](https://user-images.githubusercontent.com/58407173/187092170-def7b687-4fbd-4d81-939d-34483a87bd36.png)
![Correct Sidebar](https://user-images.githubusercontent.com/58407173/187092189-4aa5d828-e60b-49ff-920d-a387d0db665c.png)
![Wrong Panel Icons](https://user-images.githubusercontent.com/58407173/187092175-a8f5e990-e46c-4181-b999-8af273f9512e.png)

**Comment by Elbullazul** [comment]
hello,

sorry for the late reply. Which version of Mint are you using?

**Comment by AidFTech** [comment]
No worries. I am currently running 20.3 but hope to transition to 21 once I
figure out how to convert the Windows Metacity themes to GTK (since Mint 21
only renders GTK windows).

**Comment by Elbullazul** [comment]
The Windows 7 GTK theme already supports headerbar as titlebars ([code is here](https://github.com/B00merang-Project/Windows-7/blob/master/gtk-3.0/gtk.css#L1713) if you're interested)

Most other themes have support as well

**Comment by shawny43** [comment]
There a lot of problem with your icon set and theme currently and this kind of issue still persist. I cannot saw them updated. Im running on cinnamon from Linux Mint 21

**Comment by Elbullazul** [comment]
@shawny43 I'm currently more focused on GTK themes. I will post an update when I start working on this

**Comment by Xopek-Endurance** [comment]
I made a great fix for this issue(for Mint&Cinnamon, my version - LMDE 6, Cinnamon 6.0.4).

Put files from my fork to Windows-7 icon theme folder
(https://github.com/Xopek-Endurance/Windows-7-CinnamonFix)

After it all must work as in my screenshot 
![1](https://github.com/B00merang-Artwork/Windows-7/assets/108617048/0cc854ad-2eee-4339-acd8-4adee35d5fd4)
![2](https://github.com/B00merang-Artwork/Windows-7/assets/108617048/e17b5168-dd4d-4b6f-8449-23b5afb44d14)

**Comment by Elbullazul** [comment]
@Xopek-Endurance would you consider submitting a pull request with your changes?

**Comment by Xopek-Endurance** [comment]
There is one difficulty - Mint Update tray icon is not in icon folder of any themes (or I don't know about it?), because of this to change this icon i replaced icons in apllet system folder, in my case /usr/share/cinnamon/applets/notifications@cinnamon.org/icons
and you need to do this carefully and make a backup before doing this. 

I will attach here the folder with the icons that I replaced

[icons.tar.gz](https://github.com/B00merang-Artwork/Windows-7/files/15448958/icons.tar.gz)

**Comment by Xopek-Endurance** [comment]
> @Xopek-EnduranceНе могли бы вы рассмотреть возможность отправки запроса на внесение изменений?

Прошу прощения, но я не настолько хорошо владею Github'ом, чтобы сделать такое. Просто так программа отправить запрос мне не позволяет. Если я правильно понял, мне сначала надо сделать свою страничку здесь и туда залить все файлы? Если так, то я постараюсь этим заняться в течение месяца, но ничего не гарантирую - слишком сложно для меня

**Comment by Xopek-Endurance** [comment]
> @Xopek-Endurance would you consider submitting a pull request with your changes?

I tried to do this, I don’t know if I did it right...

**Comment by Elbullazul** [comment]
@Xopek-Endurance thanks for the submission! Just a question, did you reuse existing icons, or did you source them from elsewhere?

**Comment by Xopek-Endurance** [comment]
@Elbullazul

Yes, I reused icons from Your theme, just rename, move to other folder and conver to svg, not sure what's the best way. One exception - when i done icons of network applet and wifi, i didn't find them in Your theme then (But now I find and know they are in your theme), because of this I used network and wifi applet Icons from here: https://www.mediafire.com/file/s608qnmuvv3shuo/AUDIO+AND+NETWORK+ICONS.zip
